jwtAuthorizer/.env

```diff
@@ -1,2 +1,2 @@
-KEY_ID=4fc8121b-3304-4dca-8fd5-d3f7c2251bd6
+KEY_ALIAS=SessionKey
```

Comment:
Non serve perché nel controllo fa fede il kid presente sul token.
Al massimo di potrebbe indicare in Tags che la Key dovrebbe avere sul KMS, ma non complichiamo troppo.